modules/gltf/editor/editor_scene_importer_gltf.cpp

```diff
@@ -77,6 +81,7 @@ Node *EditorSceneFormatImporterGLTF::import_scene(const String &p_path, uint32_t
 
 void EditorSceneFormatImporterGLTF::get_import_options(const String &p_path,
 		List<ResourceImporter::ImportOption> *r_options) {
+	r_options->push_back(ResourceImporterScene::ImportOption(PropertyInfo(Variant::INT, "gltf/compatibility_version", PROPERTY_HINT_RANGE, "0,1"), 1));
```

Comment:
If we agree on the overall approach, this probably requires a bit of bikeshedding. Should maybe be an enum hint, with values like "Godot 4.1" and "Godot 4.2 or later"?

Reply:
That's a good idea.